vulkan_helpers/helper_functions.h

```diff
@@ -444,6 +451,27 @@ inline VkFormat GetSupportedDepthStencilFormat(VkInstance* instance, VkDevice* d
   return depth_stencil_format;
 }
 
+// Returns the highest supported depth format, VK_FORMAT_D32_SFLOAT >
+// VK_FORMAT_X8_D24_UNORM_PACK32 > VK_FORMAT_D16_UNORM
```

Comment:
We don't need to default to D16_UNORM. One of VK_FORMAT_D32_SFLOAT or VK_FORMAT_X8_D24_UNORM_PACK32 is guaranteed to be supported.

Reply:
We can add an assert if we really want to make sure, but the spec requires it.